tests/search_ops.rs

```diff
@@ -0,0 +1,72 @@
+mod common;
+
+#[cfg(feature = "beta")]
+mod search_tests {
+    use crate::common::initialize_sdk;
+    use ironoxide::{group::GroupOps, prelude::*, search::*};
+
+    #[tokio::test]
+    async fn create_index() -> Result<(), IronOxideErr> {
+        let sdk = initialize_sdk().await?;
+        let group_create_result = sdk.group_create(&Default::default()).await?;
+        sdk.create_index(group_create_result.id()).await?;
+        Ok(())
+    }
+
+    #[tokio::test]
+    async fn initialize_blind_index_search() -> Result<(), IronOxideErr> {
+        let sdk = initialize_sdk().await?;
+        let group_create_result = sdk.group_create(&Default::default()).await?;
+        let result = sdk
+            .initialize_blind_index_search(group_create_result.id())
+            .await?;
+        let index_tokens = result.sdk.tokenize_data("hello world", Option::None);
+        assert_eq!(index_tokens.len(), 6); //hel, ell, elo, wor, orl, rld --The numbers are random and aren't worth asserting about.
```

Comment:
and the stability of the token generation itself is already covered by tests in search-helpers, correct?

Reply:
Yes.